kubernetes/kubernetes.go

```diff
@@ -358,10 +377,20 @@ func (ka KubeAgentConfig) sendMetrics(metricSample *os.File) {
 
 	err = SendData(metricSample, ka.clusterUID, cldyMetricClient)
 	if err != nil {
+		log.Warnf(handleError(err))
```

Comment:
I think we only want to log this is handleError() return is not empty

Reply:
I'll try, but a lot of these methods have really high cyclometric complexity as it is, so if I add another if statement, it might irritate the linter.  I don't think that will happen in this case, but might need to refactor.